gradle/libs.versions.toml

```diff
@@ -69,6 +69,6 @@ plugin-android = { module = "com.android.tools.build:gradle", version.ref = "agp
 plugin-dokka = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version.ref = "kotlin" }
 plugin-grgit = { module = "org.ajoberstar.grgit:grgit-gradle", version = "5.0.0" }
 plugin-kotlin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
-plugin-mavenPublish = { module = "com.vanniktech:gradle-maven-publish-plugin", version = "0.21.0" }
+plugin-mavenPublish = { module = "com.vanniktech:gradle-maven-publish-plugin", version = "0.24.0" }
```

Comment:
The `paparazzi-javadoc.jar` is empty.

Reply:
This is a bug in the publish plugin will fix that over there.

Reply:
Will be resolved by https://github.com/vanniktech/gradle-maven-publish-plugin/pull/559, will do a 0.25.2 release after this is merged

Reply:
[92d0956](https://github.com/cashapp/paparazzi/pull/716/commits/92d09562056786a7ba1dd71465c9f13b450a2e4d)